### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by DarkJyka**

### CODE REVIEW

File: collection/CheesiestMaster; Teyas's Guide to Aile.json [hunk 1]
```diff
@@ -0,0 +1,801 @@
...
+            }
+        ],
+        "dateAdded": 1641853328,
+        "dateLastModified": 1641853330
+    },
+    "book": [
+        {
+            "name": "Teyas' Guide to Aile",
+            "id": "TeyasGuidetoAile",
+            "source": "http://potato.jdsnetwork.com/DnD/TGtA.json",
```
Comment (TheGiddyLimit) [inline collection/CheesiestMaster; Teyas's Guide to Aile.json:27]: needs to be `"TeyasGuidetoAile"`

File: collection/CheesiestMaster; Teyas's Guide to Aile.json [hunk 2]
```diff
@@ -0,0 +1,801 @@
...
+                "abbreviation": "TGtA",
+                "full": "Teyas' Guide to Aile",
+                "authors": [
+                    "CheesiestMaster#4124"
+                ],
+                "convertedBy": [
+                    "CheesiestMaster#4124"
+                ],
+                "version": "1.0",
+                "url": "https://github.com/TheGiddyLimit/homebrew",
```
Comment (TheGiddyLimit) [inline collection/CheesiestMaster; Teyas's Guide to Aile.json:16]: you can/should put `""http://potato.jdsnetwork.com/DnD/TGtA.json"` here instead

File: collection/CheesiestMaster; Teyas's Guide to Aile.json [hunk 3]
```diff
@@ -0,0 +1,801 @@
...
+                "B"
+            ],
+            "damageTags": [
+                "P"
+            ],
+            "miscTags": [
+                "MW"
+            ],
+            "hasFluff": true,
+            "tokenUrl": "https://5etools-mirror-1.github.io/img/MM/Swarm of Wasps.png?v=1.147.13",
```
Comment (TheGiddyLimit) [inline collection/CheesiestMaster; Teyas's Guide to Aile.json:630]: drop the `?v=1.147.13` from the end--it isn't going to break anything, but it doesn't do anything either (besides maybe serving a stale version of the image from your cache)

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]

File: collection/CheesiestMaster; Teyas's Guide to Aile.json (hunk 1 above)
Comment (revilowaldow) [inline collection/CheesiestMaster; Teyas's Guide to Aile.json:27]:
```suggestion
            "source": "TeyasGuidetoAile",
```

File: collection/CheesiestMaster; Teyas's Guide to Aile.json (hunk 2 above)
Comment (revilowaldow) [inline collection/CheesiestMaster; Teyas's Guide to Aile.json:16]:
```suggestion
                "url": "http://potato.jdsnetwork.com/DnD/TGtA.json",
```

File: collection/CheesiestMaster; Teyas's Guide to Aile.json (hunk 3 above)
Comment (revilowaldow) [inline collection/CheesiestMaster; Teyas's Guide to Aile.json:630]:
```suggestion
            "tokenUrl": "https://5etools-mirror-1.github.io/img/MM/Swarm of Wasps.png",
```

**Review by revilowaldow — CHANGES_REQUESTED** [review]

**Pull request closed without merging**